### ISSUE TITLE
Single quotes strings seem to not be accepted by the plugin as valid, while the game does not mind them

### ISSUE THREAD

**Issue by adriangaro** [DragonKnightOfBreeze/Paradox-Language-Support#65]
Some examples

![image](https://github.com/DragonKnightOfBreeze/Paradox-Language-Support/assets/11583810/b456371c-ee15-49b3-8e43-505d78185c1f)

![image](https://github.com/DragonKnightOfBreeze/Paradox-Language-Support/assets/11583810/28e3491e-3248-4c61-b7c6-1128f43f28a1)

Usecase is passing nested blocks of code/triggers that might contain themselves other quoted strings, since escaping quotes does not always work when passed through multiple layers of inlines.

**Comment by DragonKnightOfBreeze** [comment]
I even do not known thst single quotes can also be used to quote strings, are you tested in real game?

**Comment by adriangaro** [comment]
Yeah I have tested in game and no errors appeared in logs, and the logic seems to have worked/triggered in game, which i assume means it is valid xD

**Comment by isd-1** [comment]
in scripted_effects ?

**Comment by adriangaro** [comment]
Seems to work in inlines at least, unsure about scripted_effects, but can check and test.
